server/src/main/java/org/elasticsearch/search/aggregations/metrics/CardinalityAggregationBuilder.java

```diff
@@ -113,7 +113,7 @@ public boolean supportsSampling() {
     }
 
     @Override
-    public boolean supportsOffloadingSequentialCollection() {
+    public boolean supportsParallelCollection() {
```

Comment:
If these could not support concurrency due to the post collect issue, does it mean that they can now support concurrency perhaps?

Reply:
For `cardinality`, I think yes. But let's change this when parallel collection is enabled in the query phase?

Reply:
Yes